services/gundeck/src/Gundeck/Push/Native/Types.hs

```diff
@@ -34,6 +34,9 @@ import Gundeck.Aws.Arn
 import Gundeck.Types
 
 -- | Native push address information of a device.
+--
+-- REFACTOR: the @s@ phantom type can probably go away, too!
```

Comment:
`Address "keys"` was removed here: https://github.com/wireapp/wire-server/pull/546

This datatype was used to distinguish between addresses that _had_ encryption keys (and thus could be symmetrically encrypted) and those that could not.

Since we have now removed this, this distinction no longer makes sense and thus can (and should) be removed and described in the comment _REFACTOR: the @s@ phantom type can probably go away, too!_